src/item_pocket.cpp

```diff
@@ -1196,7 +1196,7 @@ void item_pocket::contents_info( std::vector<iteminfo> &info, int pocket_number,
             const std::vector<damage_info_order> &all_ablate = damage_info_order::get_all(
                         damage_info_order::info_type::ABLATE );
             for( const damage_info_order &dio : all_ablate ) {
-                std::string label = string_format( idx == 0 ? _( "<bold>Protection</bold>: %s: " ) : "%s: ",
+                std::string label = string_format( idx == 0 ? _( "<bold>Protection</bold>: %s: " ) : " %s: ",
```

Comment:
```suggestion
                std::string label = string_format( idx == 0 ? _( "<bold>Protection</bold>: %s: " ) : pgettext( "protection info", " %s: " ),
```

Just in case some languages don't want those spaces in translation.